CMakeLists.txt

```diff
@@ -79,8 +79,8 @@ if (IS_DIRECTORY "${PROJECT_SOURCE_DIR}/.git")
   execute_process(
     COMMAND git symbolic-ref HEAD
     OUTPUT_VARIABLE GIT_BRANCH
-    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-    ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
+    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR} ERROR_QUIET
```

Comment:
Was this changed by formatting script? It does not seem to be more clear.

Reply:
Yes, it's forced by format check. Otherwise I cannot commit this file.